spec/system/admin/valuator_groups_spec.rb

```diff
@@ -67,7 +67,10 @@
     create(:valuator_group)
 
     visit admin_valuator_groups_path
-    accept_confirm { click_button "Delete" }
+
+    confirmation = "Are you sure? This action will delete "\
+      "\"#{ValuatorGroup.last.name}\" and can't be undone."
```

Comment:
We run into so-called "flaky tests" (meaning sometimes tests fail with no apparent reason) when running database queries after the first `visit` in the test (see pull request #4475). Here `ValuatorGroup.last` would generate a database query which has a low probability (maybe about 1%?) to generate a `PG::ProtocolViolation` error when running the tests.

How about changing line 67 so we give the group a name like "Early birds"? Then we could write:

```suggestion
      "\Early birds\" and can't be undone."
```